src/theme.js

```diff
@@ -1,88 +1,56 @@
 const palette = {
 	grey: [
-		'rgb(255, 255, 255)',
-		'rgb(250, 250, 250)',
-		'rgb(246, 246, 246)',
-		'rgb(225, 225, 225)',
-		'rgb(187, 187, 187)',
-		'rgb(126, 126, 126)',
 		'rgb(51, 51, 51)',
+		'rgb(126, 126, 126)',
+		'rgb(187, 187, 187)',
+		'rgb(225, 225, 225)',
+		'rgb(246, 246, 246)',
+		'rgb(250, 250, 250)',
+		'rgb(255, 255, 255)',
 	],
 };
 
-let invertedPalette = {};
-
-Object.keys(palette).forEach(key => {
-	invertedPalette[key] = [...palette[key]].reverse();
-});
-
-function getColors(palette) {
-	return {
-		...palette,
-		background: palette.grey[0],
-		text: palette.grey[6],
-		primary: 'rgb(120, 51, 150)',
-		secondary: palette.grey[5],
-		accent: '#f8c124',
-		muted: palette.grey[3],
-		hover: 'rgb(248, 188, 229)',
-		focus: 'rgb(251, 209, 234)',
-		error: '#d0453e',
-	};
-}
-
-const colors = getColors(palette);
-
 const textStyles = {
-	xl: {
-		fontFamily: 'heading',
-		lineHeight: 'heading',
-		fontSize: ['lg', 'xl'],
-		fontWeight: ['bold', 'light'],
-	},
-	lg: {
+	heading: {
 		fontFamily: 'heading',
 		lineHeight: 'heading',
 		fontSize: 'lg',
-		fontWeight: 'normal',
-	},
-	md: {
-		fontFamily: 'heading',
-		lineHeight: 'heading',
-		fontSize: 'md',
 		fontWeight: 'bold',
 	},
 	body: {
 		fontWeight: 'normal',
 		fontFamily: 'body',
 		fontSize: 'md',
-		lineHeight: 'base',
+		lineHeight: 'body',
 		color: 'text',
 	},
-	secondary: {
-		fontSize: 'md',
-		fontWeight: 'normal',
-		lineHeight: 'base',
-		fontFamily: 'body',
-		color: 'grey.5',
-	},
-	tertiary: {
+	help: {
```

Comment:
Might be more specific:

```suggestion
	hint: {
```

Reply:
I went with `help` from `hint` since I'm using it for help texts. It feels more natural this way.

Reply:
Which texts are the help texts? Do you have a screenshot? 🤔

Reply:
![CleanShot 2020-04-14 at 09 17 18](https://user-images.githubusercontent.com/11071/79196659-c3feb100-7e30-11ea-989f-5aab18704cc9.png)

I agree it's not a help text. How would you call it?

Reply:
This one is small print I think.

Reply:
Okay I'll update the token name then.